### PR TITLE
Update jest

### PR DESCRIPTION
**Pull request by joaomilho**
Also changes the types for jest's Matcher since it changed somewhere in between and complains in some projects we consume it.

### CODE REVIEW

**Comment by joaomilho** [comment]
these failures look weird to me

**Comment by Nevon** [comment]
Azure pipelines blames downloads of Nodejs: https://github.com/microsoft/azure-pipelines-tasks/issues/11626

**Comment by joaomilho** [comment]
ok, but how to rebuild?

**Comment by Nevon** [comment]
You need an admin to do it. I just triggered a rebuild.

File: yarn.lock [hunk 1]
```diff
@@ -9,6 +9,13 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
+"@babel/code-frame@^7.8.3":
+  version "7.8.3"
+  resolved "https://artifactory.klarna.net/artifactory/api/npm/v-npm-production/@babel/code-frame/-/code-frame-7.8.3.tgz#33e25903d7481181534e12ec0a25f16b6fcf419e"
```
Comment (Nevon) [inline yarn.lock:14]: Looks like you have references to Klarna's NPM registry.

**Comment by erikengervall** [comment]
> You need an admin to do it. I just triggered a rebuild.

I’d like to suggest moving CI to github actions. It has been working well for dockest and I find it convenient having everything on the same platform.

WDYT?

File: yarn.lock (hunk 1 above)
Comment (erikengervall) [reply yarn.lock:14]: Classic

**Comment by Nevon** [comment]
Sure. The only reason we use Azure pipelines for KafkaJS is that we need to run 3 Kafka brokers + Zookeeper and they are the only ones that give us enough horsepower to do that. If you can run schema registry, zookeeper and 1 kafka broker on Github actions, go for it.

File: yarn.lock (hunk 1 above)
Comment (joaomilho) [reply yarn.lock:14]: AHHHAEHAEHAEHHEA damn

**Comment by joaomilho** [comment]
goddamn, yarn doesn't respect the rc file... gotta set the registry manually

**Pull request merged**